Apps/WebAPI/sdLitica.WebAPI/Models/Analytics/AnalyticsRequestModel.cs

```diff
@@ -12,6 +12,7 @@ namespace sdLitica.WebAPI.Models.Analytics
     /// </summary>
     public class AnalyticsRequestModel
     {
+        public string Id { get; set; }
```

Comment:
Please, don't forget to add xml commnets to all publicly visible fields, especially to those which are going to be visible in REST API